### ISSUE TITLE
Backup tgz filenames shouldn't have colons

### ISSUE THREAD

**Issue by thomas-anderson-bsl** [brightsparklabs/appcli#147]
`tar` doesn't like dealing with filenames with `:` characters.
We should remove/replace these, so that `tar tf $BACKUP.tgz` will work.

For datetime stamp, we could use something like:
```
from datetime import datetime
datetime.now().replace(microsecond=0).isoformat().replace(":", "-")
```

**Comment by al-jeyapal** [comment]
Delete the colons rather than replacing with a hyphen so that the string remains ISO8601 compliant

```
datetime.now().replace(microsecond=0).isoformat().replace(":", "")
```

**Comment by thomas-anderson-bsl** [comment]
Resolved by #172
